src/main/java/it/gov/pagopa/apiconfig/cache/util/ConfigMapper.java

```diff
@@ -133,6 +136,10 @@ public ModelMapper modelMapper() {
         .createTypeMap(ConfigurationKeys.class, ConfigurationKey.class)
         .setConverter(convertConfiguration);
 
+    mapper
+        .createTypeMap(GdeConfig.class, GdeConfiguration.class)
+        .setConverter(new ConvertGdeConfigToGdeConfiguration());
```

Comment:
Just for the sake of linearity with other mappers, can the definition of a `new ConvertGdeConfigToGdeConfiguration()` be separated into a separate variable?